### ISSUE TITLE
Ok buttons renders Smart Clause List even if text is not selected for binding

### ISSUE THREAD

**Issue by arteevraina** [accordproject/cicero-word-add-in-deprecated#116]
When enter `Template id` and `clause id` without selecting a text for binding and press `OK` It renders Smart Clause List. It should rather display a warning that text is not selected for binding.

Should I work on it @DianaLease @dselman ?

**Comment by dselman** [comment]
I think we need to step back a bit and think about what the user is trying to do when they press these buttons. I'd hold off changing this until then.

I.e. is this part of some flow to create a new template from existing text?

**Comment by arteevraina** [comment]
Yes, it is in that flow only. We are directed to the starting page of the add-in without selecting the text which I think should not happen? 
But I would follow your view on this.

**Comment by arteevraina** [comment]
@dselman Should we keep this issue open and work on it later if the need arises?
